app/src/main/java/fr/free/nrw/commons/settings/Prefs.java

```diff
@@ -7,8 +7,8 @@
     public static final String DEFAULT_LICENSE = "defaultLicense";
 
     public static class Licenses {
-        public static final String CC_BY_SA_3 = "CC BY-SA";
-        public static final String CC_BY_3 = "CC BY";
+        public static final String CC_BY_SA_3 = "CC BY-SA 3.0";
+        public static final String CC_BY_3 = "CC BY 3.0";
```

Comment:
That makes sense. This should ultimately be a PR but I'm fine with this being merged. As long as @domdomegg reads this and knows this for next time.